cmd/deploy.go

```diff
@@ -354,28 +355,9 @@ func autofillApimodel(dc *deployCmd) error {
 	return nil
 }
 
-func (dc *deployCmd) validateApimodel() (*api.ContainerService, string, error) {
-	apiloader := &api.Apiloader{
-		Translator: &i18n.Translator{
-			Locale: dc.locale,
-		},
-	}
-
-	p := dc.containerService.Properties
-	if strings.ToLower(p.OrchestratorProfile.OrchestratorType) == "kubernetes" {
```

Comment:
For the life of me I can't figure out how to reproduce a deploy command matching this validation failure that actually gets this far in the flow. Basically, I don't know how one would use MSI + `aks-engine deploy`

Reply:
it's very possible that it's outdated, this is really old code